src/lib.rs

```diff
@@ -448,14 +448,19 @@ impl SymResolver for ElfResolver {
         (self.loaded_address, self.loaded_address + self.size)
     }
 
-    fn find_symbol(&self, addr: u64) -> Option<(&str, u64)> {
+    fn find_symbols(&self, addr: u64) -> Vec<(&str, u64)> {
         let off = addr - self.loaded_address + self.loaded_to_virt;
-        let parser = self.get_parser()?;
+        let parser = self.get_parser();
+        if parser.is_none() {
+            return vec![];
+        }
+        let parser = parser.unwrap();
```

Comment:
Nit: Use `if let` instead of unwrapping?
```rust
let parser = if let Some(parser) = self.get_parser() {
  parser
} else {
  return Vec::new()
};
```